### ISSUE TITLE
Fix: Typo in FEATURES.md

### ISSUE THREAD

**Issue by akash0708** [RocketChat/Rocket.Chat#30682]
### Description:

Fixed a minor typo in the FEATURES.md page.

### Steps to reproduce:

1. 
2. 
3. 

### Server Setup Information:

- Version of Rocket.Chat Server: 
- Operating System: 
- Deployment Method: 
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

### Client Setup Information

- Desktop App or Browser Version:
- Operating System:

**Comment by ggazzo** [comment]
Thank you for collaborate, but I'm afraid this is not the standard issue we expect. Some considerations need to be taken.

Your issue doesn't describe or identify the problems, enabling others to understand the issue and make corrections. It seems more like the description of a pull request (PR) fix.

There's no need to open an issue just to have it closed by a PR; you can go directly to the PR.

**Comment by akash0708** [comment]
Hello, @ggazzo I'll keep this in mind amd raise a PR directly next time. I've been contributing to smaller open-source repositories for sometime now. I decided to shift to bigger, industry grade projects and that is when I came across Rocket.chat. I found it interesting but overwhelming to understand as well, hence I decided to start contributing in documentation while reading and understanding the codebase. Thank you for merging the PR, I hope I will be able to make better contributions soon.
